Models/RestaurantModel.java

```diff
@@ -3,11 +3,11 @@
 public class RestaurantModel {
 	
 	private String name;
-	public String price;
-	private String type;
+	public int price;
+	private int type;
 	private List<String> cusines;
 
-	public RestaurantModel(String name, String price, String type,
+	public RestaurantModel(String name, int price, int type,
```

Comment:
I changed the data types from strings to ints because enums are stored as ints in the database.  This will allow us to implement enums for the restaurant model without having to migrate the database.